nft_ingester/src/config.rs

```diff
@@ -560,11 +563,7 @@ mod tests {
         assert_eq!(args.rocks_db_path_container, "./my_rocksdb");
         assert_eq!(args.file_storage_path_container, "./tmp/file_storage");
         assert_eq!(args.pg_max_db_connections, 100);
-        assert_eq!(args.redis_accounts_parsing_workers, 20);
-        assert_eq!(args.redis_transactions_parsing_workers, 20);
         assert_eq!(args.sequence_consistent_checker_wait_period_sec, 60);
-        assert_eq!(args.account_processor_buffer_size, 250);
-        assert_eq!(args.account_processor_mpl_fees_buffer_size, 50);
```

Comment:
why remove these assertions and not replace with the same changes done to IngesterClapArgs?

Reply:
Not important parameters, It takes a long time to maintain positive tests. 
the main goal is check that the specified default parameters are taken